### ISSUE TITLE
Best way to handle math ($ or $$) in markup

### ISSUE THREAD

**Issue by jeffheaton** [lepture/mistune#301]
What is the best way to handle embedded LaTex math code, such as can be displayed by GitHub.

Usually Mistune just passes the $ and $$ through as text, but sometimes it does not.  Particularly, an expression such as:

`$ \tilde{C}_t = \tanh(W_C \cdot [\hat{y}_{t-1},x_t]+b_C) $`

Comes in from Mistune as:

```
Text: $ \tilde{C}
Emphasis: t = \tanh(W_C \cdot [\hat{y}
Text: {t-1},x_t]+b_C) $
```
Which makes sense, since Mistune is unaware of the $ escaped text, but it corrupts the math code, since the _'s do not indicate emphasis.

Also, are their any plans of supporting math encoding in the future?

**Comment by lepture** [comment]
@jeffheaton Hi, we don't have built-in math plugin yet.

**Comment by jeffheaton** [comment]
Thanks, @lepture . Is there any built in extensibility to the mistune library that I could add such a feature myself? Other than a direct "code hack", I am not seeing a way.

**Comment by lepture** [comment]
@jeffheaton check `v3` branch, mistune v3 has built-in math plugin.

**Comment by jeffheaton** [comment]
Thank you for adding that. I just added as a sponsor.

**Comment by lepture** [comment]
@jeffheaton I've just released v3.0.0rc1, it is quite stable now. I'm using v3 in production in [Typlog](https://typlog.com/).

The documentation is here: https://mistune.lepture.com/en/latest/plugins.html
